data_structures/linked_list/linked_list.py

```diff
@@ -96,7 +96,7 @@ def ll_kth_from_end(self, value):
             print('out of range')
             return False
         else:
-            for item in range(value - 1):
+            for _ in range(value - 1):
```

Comment:
This will give me in order from the head not from the end and will give the head twice at that.